Mlem/Models/Content/Post/PostModel+MenuFunctions.swift

```diff
@@ -148,6 +148,14 @@ extension PostModel {
                     await self.notifier.add(.success("\(self.post.locked ? "L" : "Unl")ocked post"))
                 }
             })
+            // TODO: 0.19 deprecation
+            if siteInformation.isAdmin || ((siteInformation.version ?? .infinity) > .init("0.19.3")) {
```

Comment:
0.18 deprecation?

If `siteInformation` is null, I think assuming the actions are unavailable would be safer. If we want to keep this for debugging on local instances, we can put a little `#debug` block into default to `.infinity` in debug mode

Reply:
> 0.18 deprecation?

The feature is only available on instances running versions released after 0.19.3, but is unavailable on 0.19.0, 0.19.1 and 0.19.2. Therefore, we can remove the check when 0.19.2 is deprecated, hence the `// TODO: 0.19 deprecation` comment.